src/components/tab/tab.njk

```diff
@@ -11,28 +11,25 @@
                 role="tab"
                 aria-selected="false"
                 data-tab="tab-2"
-                id="t-two"
-                tabindex="-1">
+                id="t-two">
             Tab two
         </button>
         <button class="rvt-tabs__tab"
                 role="tab"
                 aria-selected="false"
                 data-tab="tab-3"
-                id="t-three"
-                tabindex="-1">
+                id="t-three">
             Tab three
         </button>
         <button class="rvt-tabs__tab"
                 role="tab"
                 aria-selected="false"
                 data-tab="tab-4"
-                id="t-four"
-                tabindex="-1">
+                id="t-four">
             Tab four
         </button>
     </div>
-    <div class="rvt-tabs__panel" tabindex="0" role="tabpanel" id="tab-1" aria-labelledby="t-one">
+    <div class="rvt-tabs__panel" tabindex="0" role="tabpanel" id="tab-1" aria-labelledby="t-one" data-tab-panel="tab-1" hidden="">
```

Comment:
Just a FYI, you don't need to set the `hidden` attribute to an empty string here. It can be a `hidden` boolean attribute. You only have to set it to an empty string when setting `hidden` as a property in JavaScript.